modules/npmscripts.config.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
 		bundler: {
 			config: {
 				imports: {
+					'@liferay/document-library-preview-css': {
+						'/': '*',
+					},
 					'@liferay/frontend-js-react-web': {
```

Comment:
This sounds like the set of css bundles is known in advance. Would not this prevent third party devs to use this pattern in their code?

Reply:
Nope, because this file is ours and describes our platform. If people outside want to use it they will need to configure their bundler projects accordingly. For example, if they are using our target platform, they will need to extend it with their modules (remember that every bundler-import in this file ends up in the target platform configuration file so that users can transparently use them).

Reply:
I'm not sure I've understood your comment, I'm reading it as: "we (our platform) provides that css and it can be used by third-party devs thanks to it's part of the target platform configuration". If I read it correctly, then yes, I'm assuming that third party devs should be able to make use of the common css our platform provides in the same way our modules (will) do

I was considering the case of third party devs who want to add a new common css bundle to be used by their own code. I'm having a hard time to see how this could be achieved if their css needs to be declared in this file. Perhaps it does not have to be... I think that's what you wanted to say when you wrote "...they will need to configure their bundler projects accordingly" right?